src/components/date/date.test.jsx

```diff
@@ -0,0 +1,376 @@
+import React, {
+  createRef,
+} from 'react';
+import {
+  render,
+  screen,
+} from '@testing-library/react';
+import userEvent from '@testing-library/user-event';
+import Date from './date.jsx';
+
+function getLocaleDate(date) {
```

Comment:
I wonder if this functionality is worth having as part of the `ref` API on the component? 🤔

Reply:
I don't have a use-case. 

I was thinking that maybe this is good for a test query helper thingy. But I also found that bigmaven tests use hard-coded dates. That seems pretty straight-forward to write "2020-04-20" or "Apr 20, 2020" or "April 20" or "April 2020". It makes the tests easier to read. 

I do need this function for these tests since it dynamically uses today's date. That ensures our component works when no value is provided for most of its functionality. There is some test coverage for hard-coded dates too but it's not all of them.